platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -2099,23 +2012,16 @@ def set_lpmode(self, lpmode):
         handle = self._open_sdk()
         if handle is None:
             return False
-        try:
-            log_port_list = self.get_log_ports()
-            for log_port in log_port_list:
-                self.set_port_admin_status_by_log_port(log_port, SX_PORT_ADMIN_STATUS_DOWN)
-            self._set_sfp_admin_status_raw(SX_PORT_ADMIN_STATUS_DOWN)
-
-            result = self._set_lpmode_raw(lpmode)
-
-            self._set_sfp_admin_status_raw(SX_PORT_ADMIN_STATUS_UP)
-            for log_port in log_port_list:
-                self.set_port_admin_status_by_log_port(log_port, SX_PORT_ADMIN_STATUS_DOWN)
 
-            return result == SXD_STATUS_SUCCESS
-        except:
-            logger.log_warning("set_lpmode failed due to some SDK failure")
-            self._close_sdk()
-            return False
+        log_port_list = self.get_log_ports()
```

Comment:
Not quite understand the method name "get_log_ports", should it be "get_logical_ports"?

Reply:
fixed